components/network/ntp/Makefile

```diff
@@ -53,6 +55,7 @@ include $(WS_MAKE_RULES)/common.mk
 
 # We modified configure.ac and m4 macros
 COMPONENT_PREP_ACTION = ( cd $(@D) && PATH="$(PATH)" ./bootstrap )
+COMPONENT_POST_CONFIGURE_ACTION += ( cd $(BUILD_DIR_64)/ntpd; gmake ntp_parser.h)
```

Comment:
Please add a comment explaining why is this needed.  Thank you.

Reply:
removed